src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const KEYS = Object.values(robot);
+  const VALUES = Object.keys(robot);
+  const INVERSED_ROBOT = {};
+  const UNIQUE_KEYS = [ ...new Set(KEYS) ];
+
+  if (UNIQUE_KEYS.length !== VALUES.length) {
+    return null;
+  }
+
+  for (let i = 0; i < KEYS.length; i++) {
```

Comment:
The 'for' loop is not very readable. Consider using a 'for...of' loop or 'Array.prototype.forEach' method to iterate over the keys and values of the 'robot' object. This would make the code more readable and easier to understand.